class-coblocks.php

```diff
@@ -168,7 +168,7 @@ public function load_textdomain() {
 		 */
 		public function block_localization() {
 			if ( function_exists( 'wp_set_script_translations' ) ) {
-				wp_set_script_translations( 'coblocks-editor', 'coblocks' );
+				wp_set_script_translations( 'coblocks-editor', 'coblocks', __DIR__ . '/languages' );
```

Comment:
@EvanHerman you may wanna do `COBLOCKS_PLUGIN_DIR . 'languages'` here since that constant seems to be the pattern used elsewhere